package/MDAnalysis/analysis/base.py

```diff
@@ -40,17 +40,60 @@
 logger = logging.getLogger(__name__)
 
 
+class Results(dict):
+    r"""Container object for storing results.
+
+    Results are extend dictionaries by enabling values to be accessed by key,
+    `results["value_key"]`, or by an attribute, `results.value_key`. 
+    They store all results obatined from an analysis after the ``run`` call.
```

Comment:
obtained

Reply:
**IMPORTANT:** For the API it's important to say what the expectations for code are. Add here your API for Results. For example
* MDAnalysis does _not_ prescribe the names of the keys. This is up to the analysis class and must be documented by each analysis.
* MDAnalysis expects the following canonical set of keys to always  be present:  frames, times, .... where frames means this, times means that, etc (and yes, I know that times and frames are in AnalysisBase, I was trying to find quick examples)

Reply:
If there are expected keys then the `Results` class should know about it. If there are no prescriptions then it's fine as it is.